samples/Kconfig

```diff
@@ -16,6 +16,15 @@ config NCS_SAMPLES_DEFAULTS
 	help
 	  Use the default configuration for NCS samples.
 
+config NCS_CUSTOM_RPMSG_CHILD_IMAGE
+	bool "Disable rpmsg sample as child image"
+	default n
+	depends on SOC_NRF5340_CPUAPP
```

Comment:
This should also depend on either `802514_SER_HOST` or `RPMSG` being set.

Reply:
if it's generic, should it really depend on those two ? I know right now its only those two cases, but would it be more future proof to not have it depend on them ? I don't have a strong opinion

Reply:
If we change it to be generic I agree with you, it should not depend on anything more than what it already does :+1: